packages/bierzo-wallet/src/routes/addresses/index.e2e.spec.ts

```diff
@@ -2,28 +2,28 @@ import clipboardy from "clipboardy";
 import express, { Request, Response } from "express";
 import { Server } from "http";
 import { Browser, Page } from "puppeteer";
-import { sleep } from "ui-logic";
 
 import { closeBrowser, closeToast, createPage, getToastMessage, launchBrowser } from "../../utils/test/e2e";
-import { acceptEnqueuedRequest } from "../../utils/test/persona";
 import { withChainsDescribe } from "../../utils/test/testExecutor";
-import { registerPersonalizedAddress, waitForAllBalances } from "../balance/test/operateBalances";
-import { travelToBalanceE2E } from "../balance/test/travelToBalance";
 import {
-  copyAddress,
-  getAddressRow,
-  getLinkedAddresses,
-  getRemoveActions,
-  getStarnames,
-} from "./test/operateReceivePayment";
+  registerIovname,
+  registerName,
+  registerStarname,
+  waitForAllBalances,
+} from "../balance/test/operateBalances";
+import { travelToBalanceE2E } from "../balance/test/travelToBalance";
+import { copyAddress, getAddressRow, getIovnames, getStarnames } from "./test/operateReceivePayment";
 import {
-  openFirstStarnameForEditingE2E,
+  manageFirstIovnameE2E,
+  manageFirstNameE2E,
+  manageFirstStarnameE2E,
   travelToAddressesE2E,
   travelToBlockchainAddressesTabE2E,
+  travelToIovnamesTabE2E,
   travelToStarnamesTabE2E,
 } from "./test/travelToReceivePayment";
 
-withChainsDescribe("E2E > Receive Payment route", () => {
+withChainsDescribe.only("E2E > Addresses route", () => {
```

Comment:
should this `only` be here?

Reply:
Nope, good catch

Reply:
By the way, I haven't been able to successfully use `yarn test-single` or running a single test file by any other means. Do you know what the issue is? Or do you have another approach for running only a single e2e test file?

Reply:
hmm.. No, I used `yarn test-single <part of the path to file name>` + `.only` in case if I want to run specific section or test.